src/battle/battle_script_commands.c

```diff
@@ -1646,6 +1720,10 @@ u32 CalculateBallShakes(void *bw, struct BattleStruct *sp)
 
 #endif
     }
+
+    if(sp->item_work = ITEM_FRIEND_BALL && i >= 4) 
+        SetMonData(Battle_GetClientPartyMon(bw,sp->defence_client,0),ID_PARA_friend,200);
+
```

Comment:
same as the other friend ball check

Reply:
can delete this if the other friend ball check is implemented

Reply:
This friend ball check catches both capture rate > 255 and the regular random checks if crit capture isn't defined. I think it's better to keep this here than move it up, for completeness if anyone disables crit captures.

Reply:
my bad, you're totally right 😁